devfiles/meta.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 publisher: fbenoit
```

Comment:
I think here we can change the publisher (but it's not related to this PR)

Reply:
Updated with eclipse.